src/main/java/picard/fingerprint/FingerprintChecker.java

```diff
@@ -527,7 +526,7 @@ public IntervalList getLociToGenotype(final Collection<Fingerprint> fingerprints
                 final Fingerprint combinedFp = new Fingerprint(specificSample, f, null);
                 fingerprintsByReadGroup.values().forEach(combinedFp::merge);
 
-                final FingerprintResults results = new FingerprintResults(f, specificSample);
+                final FingerprintResults results = new FingerprintResults(f, specificSample, specificSample);
```

Comment:
it's a little odd to have both the readgroup and the sample be the sample_alias. wouldn't it be better to have the readgroup be null or the empty string in this case?